projects/adrv9009/builds.json

```diff
@@ -0,0 +1,25 @@
+{
+      "xilinx": {
+            "demo": {
+                  "flags": "",
+                  "hardware": [
+                        "adrv9009_zc706",
+                        "adrv9009_zcu102"
+                  ]
+            },
+            "dma_example": {
+                  "flags" : "NEW_CFLAGS=-DDAC_DMA_EXAMPLE",
+                  "hardware": [
+                        "adrv9009_zc706",
+                        "adrv9009_zcu102"
```

Comment:
this project can run on more combinations of hardware but it's ok as-is, we can extend later on